CHANGELOG.rst

```diff
@@ -53,6 +53,7 @@ Changed
   connection related errors, our code would first wait for this timeout to be reached (30 seconds)
   before returning error to the end user. #4834
 * Upgrade ``pymongo`` to the latest stable version (``3.10.0.``). #4835 (improvement)
+* Updated Parmiko to v2.7.1 to support new key formats #4891 (improvement)
```

Comment:
Would be nicer to note _what_ kind of keys are now supported with this update.

Reply:
#4891 in Changelog references unrelated PR. I guess #4901 should be used instead.